 l_gripper_sensor_controller:
    type: PR2GripperSensorController
    joint_name: l_gripper_joint
    accelerometer_name: l_gripper_motor
    left_pressure_sensor_name: l_gripper_motorl_finger_tip
    right_pressure_sensor_name: l_gripper_motorr_finger_tip
    close_speed: 0.02
    max_joint_effort: -1.0
    fingertip_force_limit: -1.0
    deformation_limit: 0.03
    force_lightest: 1.2
    hp_force_trigger: 0.1
    force_servo_force_tolerance: 0.3
    force_servo_velocity_tolerance: 0.001
    slip_servo_start_force: -1.0
    position_servo_position_tolerance: 0.01
    position_open: 0.09
 r_gripper_sensor_controller:
    type: PR2GripperSensorController
    joint_name: r_gripper_joint
    accelerometer_name: r_gripper_motor
    left_pressure_sensor_name: r_gripper_motorl_finger_tip
    right_pressure_sensor_name: r_gripper_motorr_finger_tip
    close_speed: 0.02
    max_joint_effort: -1.0
    fingertip_force_limit: -1.0
    deformation_limit: 0.03
    force_lightest: 1.2
    hp_force_trigger: 0.1
    force_servo_force_tolerance: 0.3
    force_servo_velocity_tolerance: 0.001
    slip_servo_start_force: -1.0
    position_servo_position_tolerance: 0.01
    position_open: 0.09